RELEASE_NOTE.md

```diff
@@ -0,0 +1,76 @@
+# Release v0.13.0
+
+
+## New Features
+
+* PAI protocol:
```

Comment:
PAI --> OpenPAI. this comment apply to the rest places in the entire .md file.

Reply:
updated